src/state/claim/hooks.ts

```diff
@@ -1,5 +1,5 @@
 import type { TransactionResponse } from '@ethersproject/providers'
-import { abi as MERKLE_DISTRIBUTOR_ABI } from '@uniswap/merkle-distributor/build/MerkleDistributor.json'
+import MerkleDistributorJSON from '@uniswap/merkle-distributor/build/MerkleDistributor.json'
```

Comment:
Not related to the PR, but why do we import from `build` folder? Does this package just happen to have a `build` folder or are we actually importing from `build`? Is this due to something else not working properly, such as `package.exports` not being fully supported?

Reply:
This is due to how our legacy packages are built. They export their abis in build folders. They do not use `package.exports`.

This is much older infra, so `exports` was probably just not considered at the time. I don't think we want to touch these - they are too foundational.